spec/commands/zrange_spec.rb

```diff
@@ -46,6 +46,10 @@
     @redises.zrange(@key, 5, -1).should == []
   end
 
+  it 'returns entire list when start is too small' do
+    @redises.zrange(@key, -5, -1).should == %w[Washington Adams Jefferson Madison]
+  end
```

Comment:
It could be worth adding two more tests that check the following cases in isolation: 
- negative start, positive stop
- positive start, negative stop